apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html

```diff
@@ -385,6 +385,56 @@ <h4 class="clearfix bd-rb">{{ localeData?.purchase_bill }}</h4>
                                         </div>
                                     </div>
                                 </div>
+                                <div class="clearfix pd-t1" *ngIf="voucherApiVersion === 2">
```

Comment:
*ngIf="voucherApiVersion === 2"

Write this ng-container which will common for all below settings

Reply:
Done